src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -69,6 +88,23 @@ case class AgoraEntity(@(ApiModelProperty@field)(required = true, value = "The n
                        payload: Option[String] = None
                         )
 
+@ApiModel(value = "Agora Search")
```

Comment:
Why do we need a separate AgoraSearch case class here? Just to avoid conflicts with the AgoraEntity companion object?

The alternative would be to use AgoraEntity and rename the companion object to be AgoraEntityMarshallers or similar, and then import it explicitly wherever (un)marshallers are needed.

Interested to here @jacarey's opinion